src/test/java/com/example/demo/web/AuthorControllerTest.java

```diff
@@ -69,7 +58,7 @@ void getAllAuthors_pagination() {
         assertThat(controller.getAllAuthors("", 1, 5))
                 .hasSize(1);
     }
-
+    // pagination
```

Comment:
Do we need this comment? The name of the test already explanse the test-type.

Reply:
not that necessary, we can delete it